packages/bruno-app/src/components/Welcome/index.js

```diff
@@ -29,7 +29,7 @@ const Welcome = () => {
     setImportCollectionLocationModalOpen(true);
   };
 
-  const handleImportCollectionLocation = (collectionLocation, enableTRanslation = true) => {
+  const handleImportCollectionLocation = (collectionLocation, enableTranslation = true) => {
```

Comment:
I'm pretty sure the `enableTranslation` parameter can be entirely removed, because it is not used by `importCollection`:

https://github.com/usebruno/bruno/blob/2b0ad29b9377ea3c2b9d7ecca190fe3ca137b7da/packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js#L1046

Reply:
Yes you're right it can be deleted entirely from handleImportCollectionLocation, I can confirm the import translation works without it.

Reply:
👍 @kaankeskin You can then also remove the unused parameter from here:

https://github.com/usebruno/bruno/blob/2b0ad29b9377ea3c2b9d7ecca190fe3ca137b7da/packages/bruno-app/src/components/Sidebar/TitleBar/index.js#L29